src/Scene.ts

```diff
@@ -947,14 +981,47 @@ export class Scene {
     // If 'follow' mode, then track the specifiec object.
     if (this.cameraMode === this.followEntityEvent) {
       // Using a hard-coded offset for now.
-      var relativeCameraOffset = new THREE.Vector3(-5,0,2);
+      var relativeCameraOffset = new THREE.Vector3(-5, 0, 2);
       this.cameraTrackObject.updateMatrixWorld();
       var cameraOffset = relativeCameraOffset.applyMatrix4(
         this.cameraTrackObject.matrixWorld);
 
       this.camera.position.lerp(cameraOffset, 0.1);
       this.camera.lookAt(this.cameraTrackObject.position);
 
+    } else if (this.cameraMode === this.thirdPersonFollowEntityEvent) {
+      // Based on https://discoverthreejs.com/book/first-steps/transformations/ ,
+      // in THREE.js we have the following coordinate system:
+      //
+      // +X - Across the camera, to the right
+      // -X - Across the camera, to the left
+      // +Y - Up relative to the camera
+      // -Y - Down relative to the camera
+      // +Z - towards the camera
+      // -Z - away from the camera
+
+      var fixedCameraOffset = new THREE.Vector3(-6, -2, 1.5);
```

Comment:
Refrain from using `var`. Use `let` instead (or `const` if it's a constant).

Reply:
Fixed in d2f28b9.